tools/mo/unit_tests/moc_tf_fe/check_info_messages.py

```diff
@@ -46,7 +46,6 @@ def arg_parse_helper(input_model,
         source_layout={},
         target_layout={},
         freeze_placeholder_with_value=None,
-        tensorflow_use_custom_operations_config=None,
```

Comment:
here we remove `tensorflow_use_custom_operations_config` option. Should we?

Reply:
Specifying this argument was not obligatory for this unit-tests. Even though this arg will be removed from `cli_parser` in different PR let's remove this trashy unnecessary arg setting?